### DISCUSSION TITLE
How did you fixed the audio? Is airdop working / bluetooth audio?

### DISCUSSION THREAD

**Discussion by DaniloCannas** [PIut02/ROG-Zephyrus-G14-GA401IV-Hackintosh#12] [Q&A]
2021 model on ventura, Idk how to fix the audio, what guide you used?
For the AX200 / AX210 for me airdrop is not working
Airpods connect but no audio

Ty

**Comment by PIut02** [comment]
First of all, there is no way for AirDrop to work properly on Intel network cards.  Regarding bluetooth, I found that it seems to be related to the loading order of kexts, I will fix it in the next update. 
Regarding the sound card issue, I actually described my repair process in Chinese in issue#2, if you can't fix it, please extract the codec under Linux and upload it, I will try to fix it.

**Comment by DaniloCannas** [comment]
Thank you for the answer,
I fixed the audio putting alcid=11 in boot args but idk if this is the best option.
Meanwhile for bluetooth (which is more important to me since I use AirPods a lot), I got an AX210 and the kexts I used are BlueToolFixup and IntelBluetoothFirmware.
So I need to order them differently in proper tree?

**Reply by PIut02** [reply]
I have made changes to the loading order in the recent update. Please check if the issue has been resolved. 
If you can successfully drive all four speakers, the built-in microphone, and headphone output on the G14 using the layout-id 11, then I believe this is the best option.

**Reply by qphat** [reply]
> Thank you for the answer, I fixed the audio putting alcid=11 in boot args but idk if this is the best option. Meanwhile for bluetooth (which is more important to me since I use AirPods a lot), I got an AX210 and the kexts I used are BlueToolFixup and IntelBluetoothFirmware. So I need to order them differently in proper tree?

I did the same as you and it worked. But when I restarted it a second time, the sound was gone

**Reply by DaniloCannas** [reply]
I fixed that buying m3 macbook pro
